experiences/homepage/homepage.test.js

```diff
@@ -113,5 +113,37 @@ describe("homepage experience", function () {
 		});
 
 	   expect(isAtActiveSnapPoint).equal(true);
+	});
+
+	it("close card by click when opened", async ()=>{
+		await scrollTo(browser, 800)
+
+		const firstCard = await browser.$(config.cards.homepage.cardSelector);
+
+		const firstCardIsInViewport = await firstCard.isDisplayedInViewport();
+
+		expect(firstCardIsInViewport).equal(true);
+
+		let isAtMinimisedSnapPoint = await isAtPercentageSnapPoint(browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised)
+
+		expect(isAtMinimisedSnapPoint).equal(true);
+
+		await dragCardBy(browser, config.cards.homepage.cardSelector, 200)
+
+		isAtMinimisedSnapPoint = await isAtPercentageSnapPoint(browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised)
+
+		expect(isAtMinimisedSnapPoint).equal(false);
```

Comment:
you can also check if the card is in viewport

Reply:
I think i misunderstood the test. Is this the close button?
![image](https://user-images.githubusercontent.com/2574275/114676392-8bc2d480-9d09-11eb-99e6-f29bdfcf1468.png)

If this is the button (which becomes the gradient area when the FC is open), when clicking into it, it changes the FC to the initial state, but still in viewport. Is this the close button? Or something else I'm missing?

Reply:
thats the close button, try it first as a user, see how the card behaves, and then try to reproduce it in the code

Reply:
Is this the right interaction? https://www.loom.com/share/88f502a1cffd42f28b17c2ddf4a6331c

Reply:
Clealy the close button is not visible so maybe that interaction is disabled in the flowcards.json. 
Try to find a experience with the close button enabled

Reply:
ah, no, now I see it
you have to scroll in the card to make it visible
and you'r right, it just goes back to initial state

Reply:
I'm misunderstanding the Spreadsheet with experiences. This shows it has a close button right? 
![image](https://user-images.githubusercontent.com/2574275/114677970-16f09a00-9d0b-11eb-82c7-67c254c532ad.png)

Reply:
> ah, no, now I see it
> you have to scroll in the card to make it visible
> and you'r right, it just goes back to initial state

Good catch. I guess it will be a better test if the button has NOT `opacity 0` before clicking on it? So, once the FC is fully opened, scroll a few pixels until close button is visible

Reply:
yep, a good test could be checking the styles too